packages/next-sass/readme.md

```diff
@@ -206,3 +206,41 @@ module.exports = withSass({
   }
 })
 ```
+
+
+### TypeScript
+
+You can also use `Sass` with Typescript. 
+
+```ts
+// next.config.js
+const withTypeScript = require('@zeit/next-typescript')
```

Comment:
This plugin is not needed in Next.js v9 + 🤔 

FYI, next-sass will likely to form too because of https://github.com/zeit/next.js/issues/8626

Reply:
Oh, didn't know that. Thankt you for letting me know this!